### PR TITLE
Remove slashes from log messages

### PR DESCRIPTION
**Pull request by paulb777**
Reduce confusion about deprecated Firebase pod

### CODE REVIEW

**Comment by paulb777** [comment]
Will save this for Firebase 9 and add a change log in case there is any tooling or searching on the product names in the logs.

**Review by ryanwilson — APPROVED** [review]
Can you merge this to the v9b branch, or do you want to wait until post code freeze to merge it?

**Comment by paulb777** [comment]
I'll update and merge in master after v9b merges.

**Pull request merged**
